EventHandlers/onmemberjoin.py

```diff
@@ -1,8 +1,7 @@
 import discord
 from discord.ext import commands
 from Translations.Translations import get_text as translate
-from Helpers.TranslationHelper import get_culture_from_context as culture
-
+from Commands.GuildData import get_guild_data
 notification_channel_name = 'botplayground'
 member_notification_trigger = 100
```

Comment:
Should we put both of these in a constants file? It would also make sense to make these configurable per server. This can of course be done later, it is not crucial for the functioning of this event.

Reply:
This is indeed reserved for a future feature branch :)